sale_automatic_workflow/models/sale_order.py

```diff
@@ -21,6 +21,16 @@ class SaleOrder(models.Model):
         store=True,
     )
 
+    @api.model
+    def default_get(self, fields):
+        res = super(SaleOrder, self).default_get(fields)
+        default_workflow = self.env["sale.workflow.process"].search(
```

Comment:
I think we need to check for field name to avoid extra processing for fields other than `workflow_process_id`.